code/customization/models/libSupport/keras.q

```diff
@@ -1,13 +1,21 @@
+// code/customization/models/libSupport/keras.q - Customized keras models
+// Copyright (c) 2021 Kx Systems Inc
+//
+// The purpose of this file is to include all the necessary utilities to 
+// create a minimal interface for the support of keras models. It also 
+// acts as a location to which users defined keras models are added
+
 \d .automl
 
 // @kind function
 // @category models
-// @fileoverview Fit model on training data and score using test data
-// @param data  {dict} containing training and testing data according to keys
-//   `xtrn`ytrn`xtst`ytst
+// @desc Fit model on training data and score using test data
+// @param data  {dictionary} containing training and testing data according to
```

Comment:
Params description should be capitalized and unnecessary spaces removed (applies to the whole file)

Reply:
capitalized and unnecessary space removed